File: Project 6 - Ruby to Java Conversion/Proj6Ahmed/Room.java
//Faizan Ahmed
//Project 6 - Room Class
//Nov 12, 2015

public class Room{
	
	//defining instance variables
	private String name;
	private String description;
	
	//constructor - initialize
	public Room(String theName, String theDesc){
		name = theName;
		description = theDesc;
	}
	
  //getters methods
	public String getName(){ return name; }
	public String getDescription() { return description; }
	
	//toString method
	public String toString(){
		return "The " + name + "\n" + description + "\n";
	}
	
}

File: Project 6 - Ruby to Java Conversion/white-house-tour/Room.rb
class Room
	attr_reader :name, :description
	
	def initialize the_name, the_description
		@name = the_name
		@description = the_description
	end
	
	def to_s
		return "The " + @name + "\n" + @description + "\n"
	end
	
end